DFC.ServiceTaxonomy.ContentApproval/Manifest.cs

```diff
@@ -0,0 +1,11 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Content Approval",
+    Author = "National Careers Service",
+    Website = "https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-editor",
+    Version = "0.0.2",
+    Description = "Enables a content approval workflow and related permissions",
+    Category = "Content Management",
+    Dependencies = new string[] {}
```

Comment:
We need to list the dependencies, so that the module gets initialised at the right time (with all its dependencies in place), e.g. `OrchardCore.Contents`.